regform.html

```diff
@@ -0,0 +1,172 @@
+<!doctype html>
+<html>
+<head>
+    <title> Registration Form</title>
+    <style>
+         #basicuserinfo {
+          background-color: white;
+          border: 3px solid rgb(0, 183, 255);
+          border-radius: 5px;
+          border-bottom: 1px solid rgb(0, 183, 255);
+          padding: 5px;
+          width: 50%;
+        }
+        #addressinfo {
+          background-color: white;
+          border: 3px solid rgb(0, 183, 255);
+          border-radius: 5px;
+          border-top: 1px solid rgb(0, 183, 255);
+          border-bottom-style: none;
+          padding: 5px;
+          width: 50%;
+        }
+        #preferences {
+          background-color: white;
+          border: 3px solid rgb(0, 183, 255);
```

Comment:
`border: 3px solid rgb(0, 183, 255);
  border-top: 1px solid rgb(0, 183, 255);
  border-right: 1px solid rgb(0, 183, 255);
  border-bottom: 1px solid rgb(0, 183, 255);`
  
  This looks too complicated. You don't need to overwrite all props if you want to change only the border-width.
  You can replace these 4 lines with:
  `border: 1px solid rgb(0, 183, 255);
    border-left-wdth: 3px ;`

Reply:
Thank you, I replaced those 4 lines with the ones you sent.